pkg/network/tracer/tracer_test.go

```diff
@@ -1069,14 +1072,22 @@ func (s *TracerSuite) TestDNSStats() {
 	t := s.T()
 	cfg := testConfig()
 	cfg.CollectDNSStats = true
-	cfg.DNSTimeout = 1 * time.Second
+	cfg.DNSTimeout = 5 * time.Second
```

Comment:
We should not set this so the default gets used.

Reply:
sure, looks like default is 15s